app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/LoginActivity.java

```diff
@@ -18,7 +18,7 @@
     TextView registerLink;
 
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    public void onCreate(Bundle savedInstanceState) {
```

Comment:
@mozgaanna this should be protected not public, you won't ever call this method, this will be always called by the Android system